### PR TITLE
Ex03 branch

### PR DESCRIPTION
**Pull request by Sebbek8**
Hope everything is as it should be

### CODE REVIEW

File: exersices/ex03_project/test_median.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    assert data == [1, 5, 7, 9, 15]
+
+
+def test_tuples_and_lists():
+    liste = [1, 4, 7]
+    assert median(tuple(liste)) == 4
+    assert median(liste) == 4
+
+
+test_tuples_and_lists()
```
Comment (ahmedalbuni) [inline exersices/ex03_project/test_median.py:55]: why do you have a call to this test function here?

**Review by ahmedalbuni — APPROVED** [review]
good job!

**Comment by Sebbek8** [comment]
I had it there when i tested something, then i forgot to remove it. Has no other purpose really

Mvh
Sebastian Kihle

On 6 Oct 2019, at 00:22, ahmedalbuni <notifications@github.com<mailto:notifications@github.com>> wrote:


@ahmedalbuni approved this pull request.

good job!

**Comment by ahmedalbuni** [comment]
I see, no problems
the change had been already approved, you may merge it!

Regards,
Ahmed

**Pull request merged**